sklearn/gaussian_process/tests/test_kernels.py

```diff
@@ -186,6 +187,27 @@ def test_kernel_stationary(kernel):
     assert_almost_equal(K[0, 0], np.diag(K))
 
 
+@pytest.mark.parametrize('kernel',  kernels)
+def test_kernel_input_type(kernel):
+    # Test whether kernels is for vectors or structured data
+    if isinstance(kernel, Exponentiation):
+        assert_equal(kernel.requires_vector_input(),
+                     kernel.kernel.requires_vector_input())
+    if isinstance(kernel, KernelOperator):
+        assert_equal(kernel.requires_vector_input(),
+                     np.any([kernel.k1.requires_vector_input(),
```

Comment:
why not just use `or` instead of `np.any`??

Reply:
Changed.